### PR TITLE
Inital update to 1.11.2, return of the tabula and techne modules

### PR DESCRIPTION
**Pull request by ichttt**
This is an update to 1.11.2, all copyright belongs to you (as most of the modules code is from you). Those were reimplemented because of a hats port I'm about to finish

### CODE REVIEW

**Comment by iChun** [comment]
Ahhh, I wish you could have let me know beforehand that you were gonna do this. 

I'm aiming on getting all of my mods for 1.10.2 before I update them to 1.11.2. I also have some other plans for my mod updates at the moment so for it would have been nice if this was in a different branch or something, but for Hats, I was actually planning on using a slightly different system for their rendering, the one found in Googly Eyes.

It won't be likely that I'll pull this in though, but I will use this for reference so I won't close it (also just in case anyone else tries to port the mods). I do appreciate the effort though!

File: src/main/java/me/ichun/mods/ichunutil/client/gui/config/window/WindowSetNestedIntArray.java [hunk 1]
```diff
@@ -151,12 +151,7 @@ public void elementTriggered(Element element)
                 {
                     if(e instanceof ElementTextInputNumber)
                     {
-                        ArrayList<ElementTextInputNumber> nums = elementMap.get(e.id - (e.id % 100));
-                        if(nums == null)
-                        {
-                            nums = new ArrayList<ElementTextInputNumber>();
-                            elementMap.put(e.id - (e.id % 100), nums);
-                        }
+                        ArrayList<ElementTextInputNumber> nums = elementMap.computeIfAbsent(e.id - (e.id % 100), k -> new ArrayList<ElementTextInputNumber>());
```
Comment (iChun) [inline src/main/java/me/ichun/mods/ichunutil/client/gui/config/window/WindowSetNestedIntArray.java]: Oooh, Lambdas, neat!

File: build.gradle [hunk 2]
```diff
@@ -58,23 +59,5 @@ jar {
 	}
 }
 
-// deobf jars
```
Comment (iChun) [inline build.gradle]: Why were these removed?

Comment (ichttt) [inline build.gradle]: The deobf jar was removed because if I used it for compiling with hats it gave me an unusable jar when running gradlew build and a obf jar works perfectly fine in a dev environment. The API jar was not supposed to be removed, mistake on my side

**Comment by iChun** [comment]
I wonder what this means ;)
https://github.com/iChun/iChunUtil/commit/fdff106dff4046af25b24cd13c8549adefe876fb

**Comment by ichttt** [comment]
Closing in because of rebase so you can more easily review later ;)
